### ISSUE TITLE
Reading: On the Transformaion of Latent Space in Fine-Tuned NLP Models

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#257]
## 0. Paper
 - paper: [link](https://aclanthology.org/2022.emnlp-main.97/)
 - EMNLP2022

## 1. What is it?
In this paper, authors further evaluated the methods they proposed in the past (https://aclanthology.org/2022.naacl-main.225) with multiple frameworks.

## 2. What is amazing compared to previous works?
Previous works try to analyze fine-tuned representations using supervised classification tasks.
 - https://aclanthology.org/P17-1080/
 - https://arxiv.org/abs/1812.09355

Recently, these authors proposed a method for analyzing a relationship between fine-tuned representations and human-defined information (part-of-speech, morph, or chunk)
 - https://aclanthology.org/2022.naacl-main.225

In that work, they proposed an **alignment score**, which evaluate how many concept $C_2$ (e.g. `NN: noun, singular or mass` tagged words) contains words $w$ from concept $C_1$ (e.g. each cluster from fine-tuned LM).
![スクリーンショット 2023-05-01 13 24 01](https://user-images.githubusercontent.com/45454055/235406360-be17fbb6-8acb-443d-b11a-12c8a79d0141.png)
![スクリーンショット 2023-05-01 13 24 39](https://user-images.githubusercontent.com/45454055/235406408-60ba4b9d-b4ce-42ef-b540-1a79dacf9495.png)
In the experiment, they evaluate **how many concepts (human-defined) fine-tuned model can align ($\theta = 0.9$)**
Figure 2 from the paper shows that the rate of aligned concepts
$$\frac{Number\ of\ concepts\ Eq.1 = 1}{Number\ of\ concepts}$$
![スクリーンショット 2023-05-01 13 35 31](https://user-images.githubusercontent.com/45454055/235407172-30139bc1-1cb6-442c-b4e3-7a26e25a9f4b.png)

This paper is an upgraded version of the above paper.

## 3. Where is the key to technologies and techniques?
![スクリーンショット 2023-05-01 13 08 38](https://user-images.githubusercontent.com/45454055/235405428-0d815567-053c-4bf2-92ed-f75eceacca93.png)

They adapt concept matching score (https://aclanthology.org/2022.naacl-main.225) for
 - Embedding space (clusters): pretrained vs. fine-tuned
 - Human-defined space: pos, morph, chunk labels
 - Task-specific space: positive/negative labels, NLI

### 4.1 Embedding Space (clusters)
![スクリーンショット 2023-05-01 13 09 05](https://user-images.githubusercontent.com/45454055/235405456-c9515899-6281-4032-839e-66a9af4c6114.png)

Figure 3 shows that:
 - the lower layers of the model **retain** the general language concepts learned in the base model
 - the upper layers **learn task-specific concepts**.

### 4.2 Human-defined (pos, morph, chunk)
![スクリーンショット 2023-05-01 13 09 32](https://user-images.githubusercontent.com/45454055/235405486-56d84068-0481-4dd8-b0e1-a14880837c1a.png)

Figure 4 shows that models forget the POS information in the upper layers (POS is less important for sentence classification tasks).

### 4.3 Task-specific (positive / negative)
![スクリーンショット 2023-05-01 13 09 48](https://user-images.githubusercontent.com/45454055/235405501-54cc59f2-3dd9-4cf3-93d1-5439786594aa.png)

Figure 5 shows that models learn positive / negative tags in their upper layers.

## 5. Is there a discussion?

From Figures 3, 4, and 5, only the ALBERT model behaves differently from other models.
They concluded that the cause was the Cross-Layer Paremeter Sharing possessed by ALBERT.
